hocon.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name        = 'hocon'
-  s.version     = '1.1.2'
+  s.version     = '1.1.2-SNAPSHOT'
```

Comment:
FWIW, `X.Y.Z(-[a-zA-Z0-9-\.]+)` represents a "pre-release" version in Ruby (eg `1.1.3-alpha`, `1.1.3-rc1`).